app/components/agenda/agenda-header.js

```diff
@@ -19,6 +21,20 @@ export default class AgendaHeader extends Component {
 
   @tracked loadingMessage = null;
   @tracked showLoadingOverlay = false;
+  @tracked isFinalAgenda = false;
+
+  constructor() {
+    super(...arguments);
+
+    this.loadAgendaIsFinal.perform();
+  }
+
+  @task
+  *loadAgendaIsFinal() {
+    const meeting = yield this.args.meeting;
+    const agenda = yield meeting.agenda;
+    this.isFinalAgenda = isPresent(agenda?.get('id'));
+  }
```

Comment:
Since you're awaiting in the previous line via the `yield` keyword, Ember returns you an actual object instead of a proxy, so using `.get()` is no longer necessary. You could write this as `agenda?.id`.